index.html

```diff
@@ -0,0 +1,149 @@
+        <p class="header-place-text"> @ Chena Art House, Nkwashi, Lusaka! </p>
+      </section>
+    </header>
+
+    <main>
+    <section class="main-program">
+      <div class="container-fluid">
+        <h2> Main Program </h2>
+        <hr class="orange-dividing-line-program mx-auto">
+          <div class="desktop-outline-grid-program-container">
+          <div class="grid-program-container">
+            <img src="./images/lecture.png" class=" program-image-icons" alt="illustration showing lecturer presenting">
+            <div class="program-description-title">
+              Lecture
+            </div>
+            <div class="program-description-text">
+              Listen to the speakers from various countries about the messages on community building and innovation
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/exhibition.png" class="program-image-icons" alt="illustration showing an exhibition">
+            <div class="program-description-title">
+              <p>CC Exhibition </p>
+            </div>
+            <div class="program-description-text">
+              Appreciate various creators showing off their locally brewed art pieces.
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/forum.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Forum
+            </p>
+            <p class="program-description-text">
+              Have the time to share your thoughts and opinions with experts for each topic.
+            </p>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/workshop.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Workshop
+            </p>
+            <p class="program-description-text">
+              Try creating your own creative/ technological work in groups based on your interests.
+            </p>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/ignite.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Nkwashi Ignite
+            </p>
+            <p class="program-description-text">
+              Get opportunities to network with atendees, affliliates from around the world and the presenters/ authors.
+            </p>
+          </div>
+        </div>
+        <a id="join-conference" class="join-button p-2" href="#" role="button"> JOIN THE NKWASHI CONFERENCE </a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h2 class="text-center speaker-title"> Featured Speakers</h2>
+      <hr class="orange-dividing-line-about mx-auto">
```

Comment:
- [ ] Please fix the images on featured-speakers section. On tablet screen(768px) they look disproportionate.
**Screenshot. **👇
<img src='https://user-images.githubusercontent.com/33323077/125158367-a4a42000-e168-11eb-9e81-ea045f9f2c27.png' width='300px'/>